react/ConnectedUIRouter.tsx

```diff
@@ -27,7 +27,9 @@ export function ConnectedUIRouter({
     // services plugin is necessary for UIRouter to function
     router.current.plugin(servicesPlugin);
     // apply all the plugins that are passed via props
-    plugins.forEach(plugin => router.current.plugin(plugin));
+    (plugins || []).forEach(
```

Comment:
This allows `plugins` to be optionally set via  setting directly on the `router` instance itself in the same way that `states` can be set via props or directly on the instance.